lib/minify.js

```diff
@@ -107,7 +107,12 @@ function minify(files, options) {
                 properties: false,
                 reserved: [],
                 toplevel: false,
+                regex: null,
             }, true);
+            if (options.mangle.regex !== null &&
+                options.mangle.regex.constructor.name != "RegExp") {
+                throw "Input is not a RegExp";
```

Comment:
I wouldn't worry about checking for `RegExp` here, plus you shouldn't be throwing a `String`. So I'd suggest just drop this altogether.